### PR TITLE
Create ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by DawidSAG

Ticket: [MTM-60191](https://cumulocity.atlassian.net/browse/MTM-60191)
Original PR: [7240](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7240)

[MTM-60191]: https://cumulocity.atlassian.net/browse/MTM-60191?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Change label of the position input in the dashboard settings (#7240)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md]: The Claude AI suggested to use the following title:
```suggestion
title: Rename "Position in navigator" field to "Position in tabs" in dashboard settings
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60191
+version: 1021.0.0
+---
+Change label of the position input in the dashboard settings (#7240)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md]: The Claude AI suggested to use the following body:
```suggestion
The "Position in navigator" field in dashboard settings is now labeled as "Position in tabs" for better clarity.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@DawidSAG Please check this PR

**Review by rahulc8y — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60191
+version: 1021.0.0
+---
+The "Position in navigator" field in dashboard settings is now labeled as "Position in tabs" for better clarity.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md]:
```suggestion
For better clarity, the **Position in navigator** field in the dashboard settings has been renamed to **Position in tabs**.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Rename "Position in navigator" field to "Position in tabs" in dashboard settings
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md]:
```suggestion
title: Renamed "Position in navigator" field in the dashboard settings
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**